doc/Manuals/ug-ss_x-road_6_security_server_user_guide.md

```diff
@@ -1852,3 +1854,96 @@ And the following will allow none:
 [configuration-client]
 allowed-federations=xe-test, all, none, ee-test
 ```
+
+## 19 Migrating to Remote Database Host
+
+Since version `6.22.0` Security Server supports using remote databases. In case you have an already running Security Server with local database, it is possible to migrate it to use remote database host instead. The instructions for this process are listed below.
+
+1. Shutdown X-Road processes.
+
+    ```
+    systemctl stop "xroad*"
+    ```
+
+2. Dump the local databases to be migrated. You can find the passwords of users `serverconf`, `messagelog` and `opmonitor` in `/etc/xroad/db.properties`. Notice that the versions of the local PostgreSQL client and remote PostgreSQL server must match.
+
+    ```
+    pg_dump -F t -h 127.0.0.1 -p 5432 -U serverconf -f serverconf.dat serverconf
+    pg_dump -F t -h 127.0.0.1 -p 5432 -U messagelog -f messagelog.dat messagelog
```

Comment:
It would be good to add a reminder that on a busy system the messagelog database can be quite large --> dump and restore can take considerable amount of time and disk space.